startup.sh

```diff
@@ -1,12 +1,14 @@
 #!/usr/bin/env bash
 
+nginx -s reload
```

Comment:
Should we execute `nginx -t` to test the nginx config files  and show the testing log before `nginx -s reload`